redbeat/schedulers.py

```diff
@@ -127,7 +127,7 @@ def get_redis(app=None):
                 decode_responses=True,
                 sentinel_kwargs=redis_options.get('sentinel_kwargs'),
             )
-            connection = sentinel.master_for(redis_options.get('service_name', 'master'))
+            connection = sentinel.master_for(redis_options.get('service_name', 'master'), db=redis_options.get('db', 0))
```

Comment:
I would have thought https://github.com/sibson/redbeat/pull/204/files#diff-b682cdf2618c4de54811bd9f83bc84f0287f08ea26b853fa9b29d61b20726372R126 would be sufficient to select the DB.  Can you confirm passing twice is necessary.

Reply:
@sibson 
The link you provided is for this exact PR. However if you are referring to the db option provided at line 126 of this file (unchanged from the original), then it isn't enough. That section is establishing a connection to the Sentinel whereas this is establishing a connection to the Redbeat Redis DB. It was noticed that when the Redbeat DB was not DB 0 on the Redis node, the connection would get established to the DB 0 on the Redis node and then fail to find the Redbeat schedule and job details.